docs-wiki/Support-escalations.md

```diff
@@ -0,0 +1,43 @@
+<!-- markdownlint-disable MD041 -->
+
+This document outlines the high-level support escalation process. The goal of this document is to raise awareness of contributors' roles and responsibilities within the support process. This document is not intended to offer a detailed, end-to-end process for how toolkit consumers submit and resolve support requests.
+
+On this page:
+
+- [🙋‍♀️ Support requests](#-support-requests)
+- [🔎 Escalation process](#-escalation-process)
+
+---
+
+## 🙋‍♀️ Support requests
```

Comment:
@arthurclares Should we add anything about CSU here?

Reply:
We can inform customers that they have support for using the toolkit through their unified contract. Although we won’t mention specific offers, they can always reach out to their Customer Success Account Manager (CSAM) for further assistance.

Reply:
I agree that support **can** be provided via their Unified contract, but this should not be presented as a guarantee. After all, we make it clear, in our [Support Policy](https://github.com/microsoft/finops-toolkit/blob/dev/SUPPORT.md), that users are normally restricted to filing GH issues,

Reply:
Ultimately, I want to get this detail (and more) into the doc site. SUPPORT.md would ultimately link there so it's all in one place.

Reply:
We'll address this separately.